.travis.yml

```diff
@@ -21,70 +21,123 @@ before_install:
+    - script: make verify
+      arch: arm64
+      name: "check if the vendor is up to date on arm64"
 
     - stage: "Test lint"
       script: make cloud_lint
-      name: "cloud_lint"     # names the eighth Tests stage job
+      name: "cloud_lint"
     - script: make bluetoothdevice_lint
       name: "bluetooth_device_lint"
     - script: make keadm_lint
       name: "keadm_lint"
     - script: make edge_lint
       name: "edge_lint"
 
-    - stage: "Test Build"                # naming the Tests stage
-      script: make 
-      name: "builds cloud and edge components"            # names the first Tests stage job
+    - stage: "Test Build"
+      script: make
+      arch: amd64
+      name: "builds cloud and edge components on amd64"
+    - script: make
+      arch: arm64
+      name: "builds cloud and edge components on arm64"
     - script: make bluetoothdevice
-      name: "build bluetooth_device"
+      arch: amd64
+      name: "build bluetooth_device on amd64"
+    - script: make bluetoothdevice
+      arch: arm64
+      name: "build bluetooth_device on arm64"
+    - script: make edge_small_build
+      arch: amd64
+      name: "build small edge version on amd64"
     - script: make edge_small_build
-      name: "build small edge version"
+      arch: arm64
+      name: "build small edge version on arm64"
     - script: make edge_cross_build
-      name: "cross build edge"     # names the fifth Tests stage job
+      name: "cross build edge"
     - script: make edge_cross_build_v7
-      name: "cross build edge for armv7"     # names the fifth Tests stage job
+      name: "cross build edge for armv7"
     - script: make edgesite_cross_build
-      name: "cross build edgesite"    # names the fifth Tests stage job
+      name: "cross build edgesite"
     - script: make edgesite_cross_build_v7
-      name: "cross build edgeitefor armv7"     # names the fifth Tests stage job
+      name: "cross build edgeite for armv7"
 
     - stage: "unit tests"
       script: make edge_test
-      name: "unit test edge"     # names the third Tests stage job
+      arch: amd64
+      name: "unit test edge on amd64"
+      #- script: make edge_test
+      #- arch: arm64
+      #- name: "unit test edge on arm64"
     - script: make edge_integration_test
-      name: "integration test edge"     # names the fourth Tests stage job
+      arch: amd64
+      name: "integration test edgeo on amd64"
+      #- script: make edge_integration_test
+      #- arch: arm64
+      #- name: "integration test edge on arm64"
+    - script: make cloud_test
+      arch: amd64
+      name: "unit test cloud on amd64"
     - script: make cloud_test
-      name: "unit test cloud" 
+      arch: arm64
+      name: "unit test cloud on arm64"
 
     - stage: "e2e tests"
       script: make e2e_test
-      name: "e2e_test"     # names the seventh Tests stage job
+      arch: amd64
+      name: "e2e_test on amd64"
+      # - script: make e2e_test
+      # - arch: arm64
```

Comment:
E2e test in arm OS need modify the config like pause container image.

Reply:
I think this is an important test and we should modify the configuration / test to test both arm64 AND amd64.

Reply:
Yes, sure, I think @chendave is still testing.

Reply:
Do we need arm32?

Reply:
let's go with arm32 first, I am confident with arm32. :) 
this will going to unmasked when the issue fixed, this PR is the ground work for the feature.